apps/cyberstorm-storybook/stories/components/Switch.stories.tsx

```diff
@@ -22,10 +22,41 @@ const SameStateSwitchTemplate: StoryFn<typeof Switch> = () => {
     </div>
   );
 };
+const SwitchLabelIdTemplate: StoryFn<typeof Switch> = () => {
+  const [state, setState] = useState(false);
+  return (
+    <div>
+      <label htmlFor="my-switch">My switch</label>
+      <div>
```

Comment:
This `div` doesn't really do anything so it's not needed.

Reply:
The earlier comment hasn't been addressed?

Reply:
No it's!